modified header to look like mock

EtoKruto opened the pull request:
'modified header to look like mock - used user, if defined, to show icons: inbox, notification and profile'

esteban-gs commented on client/src/components/Header.tsx:
```diff
@@ -95,6 +96,17 @@ const useStyles = makeStyles()((theme: Theme) => ({
   },
+  menuButton: {
+    color: '#323232',
+    fontFamily: 'Poppins',
```
Font family is set at the theme level, so, you could remove it.
```suggestion
```

esteban-gs commented on client/src/components/Header.tsx:
```diff
@@ -95,6 +96,17 @@ const useStyles = makeStyles()((theme: Theme) => ({
+  menuButton: {
+    color: '#323232',
+    fontFamily: 'Poppins',
+    fontSize: '1rem',
+    fontStyle: 'normal',
+    fontWeight: 400,
+    lineHeight: 'normal',
+    letterSpacing: '0.00938rem',
+    width: '100%',
+    margin: '0 5px 0 5px',
```
Please check if any of these aren't already set at the theme level.

esteban-gs commented on client/src/components/Header.tsx:
```diff
@@ -171,164 +183,189 @@ function Header() {
+                endIcon={isExchangeMenuOpen ? <ExpandLessIcon /> : <ExpandMoreIcon />}
+                sx={{
+                  textTransform: 'capitalize',
+                  color: '#323232',
```
It'll be more maintainable to get the primary/secondary color from the theme definition.

`theme.palette.primary.main`

Theme is imported from `src/theme/index.tsx`

esteban-gs reviewed (approved):
@alexislcstone , Thanks for working on this! I added a couple of comments for improvements.

I think we could also benefit from splitting up `Header` into smaller components, but up to you if want to tackle it in this PR.

EtoKruto commented:
> @alexislcstone , Thanks for working on this! I added a couple of comments for improvements.
> 
> I think we could also benefit from splitting up `Header` into smaller components, but up to you if want to tackle it in this...

Totally think it is a good idea but rather leave it for another sprint or for someone else to split, there is some refactoring too I can see. @esteban-gs I addressed the questions and will be pushing to main here shortly.
